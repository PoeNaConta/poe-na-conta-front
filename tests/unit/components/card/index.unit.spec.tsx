import React from 'react';
import Card from '@components/card';
import { render } from '@testing-library/react';

describe('Card', () => {
  it('should render card component', () => {
    const { container } = render(<Card></Card>);

    expect(container).toMatchSnapshot();
  });
});
