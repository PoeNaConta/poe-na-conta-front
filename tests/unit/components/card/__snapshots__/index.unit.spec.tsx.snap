// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Card > should render card component 1`] = `
<div>
  <div
    class="lds--card"
  />
</div>
`;
